contents/handbook/engineering/posthog-com/roadmap.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Roadmap
+---
+
+## Creating a new roadmap item
+
+- Log in to Squeak!
+- Click Roadmap in the sidebar
+- Click New
+
+### Roadmap fields
+
+#### Title*
+The title of the roadmap item. Self-explanatory.
+
+#### Description
+A brief description of what the roadmap item intends to accomplish.
+
+#### Projected completion date / Date completed
+The projected completion/completion date of the roadmap item. If the “Complete” checkbox is checked, this field label will change from “Projected completion date” to “Date completed”.
+
+This field also controls where the roadmap item appears on the roadmap page. If no date is present, it will appear under “Under consideration”. If a projected completion date exists, it will appear under “In progress”. If a completed date is added, it will appear under “Recently shipped”
```

Comment:
```suggestion
This field also controls where the roadmap item appears on the roadmap page. 

- If no date is present, it will appear under “Under consideration”. 
- If a projected completion date exists, it will appear under “In progress”. 
- If a completed date is added, it will appear under “Recently shipped”
```